osi_hostvehicledata.proto

```diff
@@ -51,7 +51,7 @@ message HostVehicleData
     //
     optional BaseMoving location_rmse = 2;
 
-    // The basic parameters of the vehicle.
+    // The basic parameters and overall states of the vehicle.
```

Comment:
(very minor) Personally I don't think you need to (or should!) update this comment.  It sets a trend that all subfields in the `VehicleBasics` message have to be listed in any comments associated with it which is duplication and makes maintenance harder.  Don't feel strongly though if you'd like to leave it in.